HealthcareSystem/PatientWebApp/DTOs/PatientDTO.cs

```diff
@@ -29,5 +29,31 @@ public class PatientDTO
         public string Password { get; set; }
 
         public PatientDTO() { }
+
+        public PatientDTO(string name, string surname, string jmbg, GenderType gender, DateTime dateOfBirth, 
+                          string phone, int countryId, string countryName, int cityZipCode, string cityName, 
+                          string homeAddress, BloodType bloodType, RhFactorType rhFactor, bool hasInsurance, 
+                          string lbo, string alergies, string medicalHistory, string email, string password)
+        {
```

Comment:
I am not sure really about this, is good to have too many parameters in DTO? Or maybe make some structure for some of these parameters?

Reply:
I think that PatientDTO must have a lot of parameters because it is one of the requirements when registering a patient. I think that the division into new structures would make confusion and make work more difficult. Especially because all the fields refer to the patient and I don't see meaningful wholes that I could segregate.

Reply:
We may not have understood each other properly. It's okay to have a lot of parameters. I think to the more readable and reusable structure of code, this too hard to read. The division into new structures would make cleaner and better code, that is all point about clean code.

Look how I propose refactoring in [this pr](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/61#issuecomment-731607853). I extracted some similar properties, and after that, you can reuse those of structure and validate only them if you want.

I’m not sure if is a practice to use smarter structure in dto's. And if we don't use that in dto's. Then that is okay. But if it is a practice to have some smarter structure in dto's, we should look at [this pr](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/61#issuecomment-731607853) and make similar to this code.

Reply:
I could refactor that in some future pull requests. For now, I would like to merge with develop, so that my team can see my tests and continue with testing.

Reply:
That it's okay for me.